runtime/include/tt/runtime/detail/ttnn.h

```diff
@@ -80,6 +80,8 @@ constexpr std::size_t kL1SmallSize = 1 << 15;
 
 std::pair<SystemDesc, DeviceIds> getCurrentSystemDesc();
 
+Tensor createEmptyTensor();
```

Comment:
I think this might be better named, `createNullTensor()`.

Reply:
fixed